nixos/modules/misc/ids.nix

```diff
@@ -275,6 +275,8 @@
       gocd-server = 252;
       terraria = 253;
       mattermost = 254;
+      quagga = 255;
```

Comment:
The Quagga daemons (zebra, ripd, ospfd, ...) run as user 'quagga' / group 'quagga' and extra group 'quaggavty'.
I've assigned static user/group IDs like all the other modules use for daemons.